go/vt/tabletserver/messager_cache.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2017, Google Inc. All rights reserved.
```

Comment:
This is the intent. For now, things are still fluid. So, I'm keeping everything in one place.
As for naming, yeah. Messenger is too overloaded. Although Messager is not proper english, it still conveys the correct meaning :).
BTW, there's a separate MessageManager type coming (one instance for each message table).